tools/bliss-to-ogg-zip.py

```diff
@@ -366,19 +368,25 @@ def main():
       if os.path.exists(dest_filename):
         print("already exists, delete: %s" % os.path.basename(dest_filename))
         os.remove(dest_filename)
-      cmd = ["ffmpeg"]
-      if args.ffmpeg_acodec:
-        cmd += ["-acodec", args.ffmpeg_acodec]  # https://trac.ffmpeg.org/ticket/2810
-      cmd += ["-i", source_filename]
-      if args.number_of_channels > 0:
-        cmd += ["-ac", str(args.number_of_channels)]
-      if start_time:
-        cmd += ["-ss", str(start_time)]
-      if limit_duration:
-        cmd += ["-t", str(duration)]
-      cmd += [dest_filename, "-loglevel", args.ffmpeg_loglevel]
-      print("$ %s" % " ".join(cmd))
-      check_call(cmd)
+      if args.no_conversion:
+        print("skip ffmpeg, copy instead (%s -> %s)" % (
+        os.path.basename(rec_filename), dest_filename[len(dest_dirname) + 1:]))
+        shutil.copy(src=source_filename,
+                    dst=dest_filename)
```

Comment:
Note: It is a bad idea to format code like this.
Better (if it fits in one line, which is the case here I think):
```suggestion
        shutil.copy(src=source_filename, dst=dest_filename)
```
Or:
```suggestion
        shutil.copy(
          src=source_filename, dst=dest_filename)
```
Or:
```suggestion
        shutil.copy(
          src=source_filename,
          dst=dest_filename)
```